scripts/accumulator-server.py

```diff
@@ -44,36 +44,44 @@
 import atexit
 import string
 import signal
+import json
 
 
 # This function is registered to be called upon termination
 def all_done():
     os.unlink(pidfile)
 
 # Default arguments
-port = 1028
-host='0.0.0.0'
+port       = 1028
```

Comment:
Python style is automatically checked by the pep8 tool. Not sure if this "cross-application" of C/C++ based style would make happy to pep8. However, we can skip this by the moment (probably the accumulator-server.py in its version before this PR is not ok...).

NTC